lingua_franca/lang/parse_fr.py

```diff
@@ -939,6 +943,9 @@ def date_found():
     if datestr != "":
         if not hasYear:
             temp = datetime.strptime(datestr, "%B %d")
+            if extractedDate.tzinfo:
+                temp = temp.replace(tzinfo=gettz("UTC"))
+                temp = temp.astimezone(extractedDate.tzinfo)
```

Comment:
What's going on here?

Reply:
Strptime does not have tzinfo, so its added automatically

Reply:
Otherwise dt comparison fails a few steps below